### PR TITLE
delete code api endpoint

### PR DESCRIPTION
**Pull request by RaviKhandavilli**
added boiler plate code for the delete code in api endpoints

### CODE REVIEW

File: api/design_api.partials.yml [hunk 1]
```diff
@@ -459,3 +459,49 @@
     summary: Update a design code
     tags:
       - designCodes
+  #------------------------------------#
+  # Delete Code Zip
+  #------------------------------------#
+  delete:
+    operationId: deleteDesignCode
+    summary: Delete a zipped design code file owned by user
```
Comment (myungjin) [inline api/design_api.partials.yml:467]:
```suggestion
    summary: Delete code of a specified version from a given design
```

File: pkg/openapi/api_design_codes.go [hunk 2]
```diff
@@ -98,6 +104,25 @@ func (c *DesignCodesApiController) CreateDesignCode(w http.ResponseWriter, r *ht
 	EncodeJSONResponse(result.Body, &result.Code, w)
 }
 
+// DeleteDesignCode - Delete a zipped design code file owned by user
```
Comment (myungjin) [inline pkg/openapi/api_design_codes.go:107]: Consider to update it based on the revised description based on the above comment.

**Review by myungjin — COMMENTED** [review]
one comment left; please consider to address it.

File: api/design_api.partials.yml (hunk 1 above)
Comment (RaviKhandavilli) [reply api/design_api.partials.yml]: Done

File: pkg/openapi/api_design_codes.go (hunk 2 above)
Comment (RaviKhandavilli) [reply pkg/openapi/api_design_codes.go]: Done

**Review by myungjin — APPROVED** [review]
Thanks. lgtm.

**Pull request merged**